src/Plugins/OrientationAnalysis/test/CMakeLists.txt

```diff
@@ -11,7 +11,6 @@ set(${PLUGIN_NAME}UnitTest_SRCS
   BadDataNeighborOrientationCheckTest.cpp
   CAxisSegmentFeaturesTest.cpp
   ConvertHexGridToSquareGridTest.cpp
-  ConvertQuaternionTest.cpp
```

Comment:
Was this test meant to be removed?

Reply:
It should not be removed. That is an active filter. Unless the actual test does not do anything, at which point the test needs to be implemented.